commands/commands.go

```diff
@@ -98,7 +97,8 @@ func runAction(actionName string, c CommandLine, api libmachine.API) error {
 
 func fatalOnError(command func(commandLine CommandLine, api libmachine.API) error) func(context *cli.Context) {
 	return func(context *cli.Context) {
-		api := libmachine.NewClient(mcndirs.GetBaseDir())
+		api := libmachine.NewClient(mcndirs.GetBaseDir(), mcndirs.GetMachineCertDir())
```

Comment:
Yes. To be honest, it has to be improved in a near future. I'm thinking about how we could pass a `MachinesDirs` struct that would be used in different places.